jdaviz/core/tools.py

```diff
@@ -330,6 +334,9 @@ def on_mouse_event(self, data):
         msg = LineIdentifyMessage(self.line_names[ind], sender=self)
         self.viewer.session.hub.broadcast(msg)
 
+    def is_visible(self):
+        return len([m for m in self.viewer.figure.marks if isinstance(m, SpectralLine)]) > 0
```

Comment:
delayed note to self... if we pass the original message on through `is_visible`, we could potentially replace this with `len(msg.marks)` (but might need to also check for message type and fallback on this implementation anyways...)